Azure-Samples/communication-services-web-calling-hero
# Update JS dependencies

Run `npm update`

## Purpose

Update dependencies to fix several vulnerabilities. This also fixes the vulnerabilities reported by GitHub's Dependabot.

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ x ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

Run the sample locally and verify all interactions - start/stop calls, join calls, screen share, enable/disable mic and camera.
